src/elements/content-sidebar/activity-feed/version/Version.scss

```diff
@@ -1,11 +1,12 @@
 @import '../../../common/variables';
```

Comment:
Can we import this in `../mixins` to avoid having to import it in every activity item stylesheet?